frontend/src/v5/ui/themes/theme.ts

```diff
@@ -268,7 +268,6 @@ export const theme = createMuiTheme({
 				paddingTop: 7,
 				paddingBottom: 22,
 				flex: 0,
-				backgroundColor: COLOR.TERTIARY_LIGHTEST,
```

Comment:
Why did we take out this?

Reply:
![image](https://user-images.githubusercontent.com/62481250/154240302-357ccdd3-522d-4e77-9bb6-799ec0c82b76.png)
It changes the error modals to look like this, and doesn't affect the way the upload modal looks (with the way it's currently coded)